src/components/UI/Controls/Navigation/Hamburger.js

```diff
@@ -54,6 +67,7 @@ function Hamburger({ onClickHandler, onKeyDownHandler, isOpen }) {
       <Bar />
       <Bar />
       <Bar />
+      <Bar />
```

Comment:
Out of curiosity: why does it take four bars to draw a burger menu that has three visible bars?

Reply:
Wow, nice :)